scripts/upgrade.py

```diff
@@ -8,6 +8,8 @@
 
 import datastore_upgrade
 
+LATEST_VERSION = 'Latest-version'
```

Comment:
You can use this for checking the status in a dict like the following:

`{'status': 'success', 'version': LATEST_VERSION, 'message': 'AppScale is currently at its latest version'}
`
or
`{'status': 'warning', 'version': LATEST_VERSION, 'message': 'AppScale is currently at its latest version'}
`